qiskit_aer/backends/aer_compiler.py

```diff
@@ -236,7 +236,8 @@ def _convert_jump_conditional(self, cond_tuple, bit_map):
             if isinstance(cond_tuple[0].var, Clbit):
                 expr = Var(bit_map[cond_tuple[0].var], cond_tuple[0].type)
             elif isinstance(cond_tuple[0].var, ClassicalRegister):
-                expr = Var([bit_map[clbit] for clbit in cond_tuple[0].var], cond_tuple[0].type)
+                reg = ClassicalRegister(bits=[bit_map[clbit] for clbit in cond_tuple[0].var])
+                expr = Var(reg, cond_tuple[0].type)
```

Comment:
I think the real trouble is that this line should never _actually_ remap the bits at all; it's a trouble with Qiskit core's classical data model that individual clbits can be remapped at all.  Registers can't really be remapped, and "proper" `Var` nodes have a complete explicit representation of "closure" so don't have the same trouble.

Assuming this passes the tests, I think we can live with it, but really, I think unless each `clbit` just maps to itself, it's _probably_ a bust circuit incoming already.